packageBackup.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "Review-Service",
+  "jest": {
+    "verbose": true,
+    "setupFilesAfterEnv": [
+      "<rootDir>src/setupTests.js"
+    ],
+    "testEnvironment": "jsdom",
+    "globals": {
+      "window": {}
+    }
+  },
+  "version": "1.0.0",
+  "main": "server.js",
+  "description": "Returns Review Components",
+  "author": "Barsha Shrestha",
+  "license": "na",
+  "engines": {
+    "node": ">=6.13.0"
+  },
+  "scripts": {
+    "webpk": "webpack -d --watch",
+    "server-dev": "nodemon server/server.js",
+    "seedData": "node database-mongoose/dataGenerator.js",
+    "test": "jest",
+    "deploy1": "aws s3 cp . s3://s3-fec-reviews --recursive",
+    "deploy2": "aws s3 sync . s3://s3-fec-reviews"
+  },
+  "resolutions": {
+    "styled-components": "^5"
+  },
+  "dependencies": {
+    "axios": "^0.21.1",
```

Comment:
I see you commented out your axios imports and added the script for the CDN. Consider removing it from your package.json if you're not using it on your front-end as an import.

Reply:
Good catch! Thanks!